src/pages/documentation/profile/index.md

```diff
@@ -0,0 +1,80 @@
+import Tabs from './tabs/index.md'
+
+# Profile
+
+You can use the Profile extension to store attributes about your user on the client. This information can be used later to target and personalize messages during online or offline scenarios, without having to connect to a server for optimal performance. The Profile extension manages the Client-Side Operation Profile (CSOP) and provides a way to react to APIs, updates user profile attributes, and shares the user profile attributes with the rest of the system as a generated event.
+
+The Profile data is used by other extensions to perform profile-related actions. An example is the Rules Engine extension that consumes the profile data and runs rules based on the profile data.
+
+<InlineAlert variant="info" slots="text"/>
+
+The Profile extension does not require any configuration.
+
+To get started with the Profile extension:
+
+1. Configure the Profile Extension in the Data Collection UI.
+2. Add the Profile extension to your app.
+3. Implement Profile APIs to:
+   * Update user attributes.
+   * Remove user attributes.
+
+## Install the Profile extension in the Data Collection UI
+
+1. In the Data Collection UI, in your mobile property, select the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Profile** extension, and select **Install**.
+3. There are no configuration settings for **Profile**.
+4. Select **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+![AEP Profile extension configuration](./assets/index/configuration.png)
+
+## Add the Profile extension to your app
+
+To add the Profile extension to your app:
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="4"/>
+
+iOS (ACP 2.x)
+
+<Tabs query="platform=ios-acp&task=add"/>
+
```

Comment:
Looks like this section is missing the React Native tab, let's include it too.
It should look like one of the other extensions, e.g. https://developer.adobe.com/client-sdks/documentation/adobe-audience-manager/
You can find the relevant names and samples here https://github.com/adobe/react-native-acpuserprofile#2-install-javascript-packages

Reply:
Updated in https://github.com/AdobeDocs/acp-mobile-sdkdocs/pull/7/commits/95e8a39b0590c8b1da359b1707e2b83a3408414b